src/types.rs

```diff
@@ -492,6 +492,8 @@ pub struct RuntimeConfig {
 	pub disable_rpc: Option<bool>,
 	pub disable_proof_verification: Option<bool>,
 	pub max_parallel_fetch_tasks: usize,
+	/// Number of concurrent queries for cell fetching
+	pub query_proof_rpc_parallel_tasks: usize,
```

Comment:
Should we add this parameter for sync client also?

Reply:
we probably should, I'll see how it goes.

Reply:
I've looked into it, since for sync_client we only fetch 8 cells, there's no need. Though, in the future, we may want to add it in case we are over the 30 cell limit (which is unlikely, but we should probably cover that case anyway).

Reply:
Agreed, no need, totally forgot we're not fetching partitions through sync...